src/Data/RFC3339String.purs

```diff
@@ -39,6 +48,17 @@ trim (RFC3339String s) =
 fromDateTime :: DateTime -> RFC3339String
 fromDateTime = trim <<< RFC3339String <<< format iso8601Format
 
+-- | Reads the locale, returning GMT (+0000) if not present.
+toLocale :: RFC3339String -> Locale
+toLocale (RFC3339String s) = Locale Nothing $ fromMaybe zero $ unsafePartial $ do
+  re <- hush $ RE.regex "([-|\\+])(\\d\\d):?(\\d\\d)$" RE.noFlags
```

Comment:
Wouldn't we need to account for `Z` here as well?

Reply:
Nope, if it's `Z`, then the regex will fail and default to an offset of `+0000`